dev/source/docs/ros-install.rst

```diff
@@ -14,11 +12,13 @@ Ubuntu 18.04 users will need to install ROS Melodic. General instructions are av
 Installing MAVROS
 -----------------
 
-Instructions for `installing MAVROS can be found here <https://github.com/mavlink/mavros/tree/master/mavros#installation>`__ but in short involve running the following command
+MAVROS is compatible with all recent versions of ROS including Kinetic, Melodic and Noetic.
+
+Instructions for `installing MAVROS can be found here <https://github.com/mavlink/mavros/tree/master/mavros#installation>`__ but in short involve running the following command.
 
 .. code-block:: bash
 
-    sudo apt-get install ros-kinetic-mavros ros-kinetic-mavros-extras
+    sudo apt-get install ros-melodic-mavros ros-melodic-mavros-extras
```

Comment:
I think that is better to default to noetic  as it support py3 !

Reply:
Fixed!